coderd/apidoc/docs.go

```diff
@@ -14314,9 +14314,6 @@ const docTemplate = `{
         "codersdk.OAuth2ProviderApp": {
             "type": "object",
             "properties": {
-                "callback_url": {
```

Comment:
Just double-checking: this API hasn't been included in a release yet, right?

Reply:
Nope, not part of any release.
It used to be behind a check for dev mode, but it's behind an experiment and dev check for a few commits now.